lib/cisco_node_utils/cmd_ref/bridge_domain.yaml

```diff
@@ -0,0 +1,44 @@
+# bridge_domain
+# bridge_domain feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k]
+
+all_bds:
+  multiple: true
+  get_command: "show bridge-domain"
+  get_value: '/^Bridge-domain (\d+) /'
+
+create:
+  set_value: "system bridge-domain add %s ; bridge-domain %s"
+
+destroy:
+  set_value: "no bridge-domain %s ; system bridge-domain remove %s"
+
+fabric_control:
+  get_command: "show bridge-domain summary"
+  get_value: '/^\w+ fabric-control [a-zA-Z0-9-_\/\s]+:\s+(\d+)/'
+  set_value: "bridge-domain %s ; %s fabric-control"
+  default_value: "0" 
+
+member_vni:
+  get_command: "show running-config bridge-domain | inc 'member vni'"
+  get_value: '/^  member vni (.*)/'
+  set_value: "%s vni %s ; bridge-domain %s ; %s member vni %s"
```

Comment:
The first one and the last one are set_value commands but the bridge-domain %s is for the context.